src/main/java/ch/njol/skript/effects/EffExecuteFunction.java

```diff
@@ -0,0 +1,72 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.function.Function;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Execute Function")
+@Description("Executes a function with the given parameters.")
+@Examples("run function \"rotate_%{_direction}%\" with {_structure-name}")
```

Comment:
I would prefer having a more in  depth example for this specific advanced feature, like add 2 functions with the possible names and params

Reply:
I'm not sure what you mean

Reply:
Here is an example which you can commit if you like :)
```suggestion
@Examples({
	"function rotate_north(structure: text):",
		"\t# code",
	"",
	"function rotate_east(structure: text):",
		"\t# code",
	"",
	"run function \"rotate_%{_direction}%\" with {_structure-name}"
})
```